jerry-core/parser/js/js-parser-internal.h

```diff
@@ -100,6 +100,18 @@ typedef enum
                                                *   CBC_PUSH_LITERAL instruction  */
 } parser_expression_flags_t;
 
+/**
+ * Initializer parsing flags.
+ */
+typedef enum
+{
+  PARSER_PATTERN_TARGET_ON_STACK = 0,         /**< assignment target is the topmost element on the stack */
```

Comment:
How can you "not pass" this flag? For let/const there is no target on the stack.

Reply:
`PARSER_PATTERN_PARSE_RHS` overrides this flag, but I'll rename these flags for easier usage.